ssh/config.json

```diff
@@ -1,11 +1,15 @@
 {
   "name": "SSH server",
-  "version": "7.1",
+  "version": "8.0",
   "slug": "ssh",
   "description": "Allows connections over SSH",
   "url": "https://github.com/home-assistant/hassio-addons/tree/master/ssh",
   "arch": ["armhf", "armv7", "aarch64", "amd64", "i386"],
   "startup": "services",
+  "ingress": true,
+  "ingress_port": 0,
```

Comment:
We do not run on host network, we can use the default ingress port

Reply:
Yep, copy/paste error simply forgot about it. Good catch!

Reply:
Fixed.